Resources/Prototypes/Voice/disease_emotes.yml

```
@@ -56,10 +66,20 @@
     components:
     - BorgChassis
   chatTriggers:
-    - yawns
+    - "chat-emote-trigger-yawn-1" # WD EDIT
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)